### ISSUE TITLE
Add map view for Android Auto

### ISSUE THREAD

**Issue by dimaguy** [mjaakko/NeoStumbler#123]
Given most of my data collection is done while driving, would it be possible to see the map getting filled using android auto instead of having it show up on the phone screen?

**Comment by mjaakko** [comment]
This sounds useful for those collecting data while driving. I don't have experience with Android Auto and no use for this myself, so I won't be doing this, at least for now

PRs are welcome if someone wants to implement this

**Comment by golden-spiral** [comment]
This would be awesome, but my knowledge is  limited regarding an implementation. It does not look too complicated from a quick search (https://www.inovex.de/de/blog/how-to-add-android-auto-functionality-to-your-existing-app/).

**Comment by hiSandog** [comment]
An Android Auto implementation should be deliberately glanceable and passive: current position/coverage progress with no small controls or interactions that encourage use while moving. It must use the approved car-app templates/categories and degrade gracefully when the host disallows arbitrary map rendering. An emulator test plus parked/driving-state behavior would help confirm data collection continues without requiring the phone display or unsafe input.

**Comment by mjaakko** [comment]
This could also be done through a third-party app after #1236 is implemented
